paillier.go

```diff
@@ -7,29 +7,28 @@ import (
 )
 
 type PublicKey struct {
-	N, G *big.Int // usually G is set to N+1
-	n2   *big.Int // the cache value of N^2
+	N *big.Int // usually G is set to N+1
```

Comment:
Comment is no longer valid. Is the G from here the one you moved to `thresholdkey.go`? Could be helpful to put this comment there as `// usually G is to PublicKey.N + 1` or something.

Reply:
Oops, I missed this comment. Good catch!